storage/onpremise/mongodb/variables.tf

```diff
@@ -1,36 +1,50 @@
-# Global variables
 variable "namespace" {
   description = "Namespace of ArmoniK resources"
   type        = string
+  default     = "default"
 }
 
-# Parameters for MongoDB
-variable "mongodb" {
-  description = "Parameters of MongoDB"
-  type = object({
-    image              = string
-    tag                = string
-    node_selector      = map(string)
-    image_pull_secrets = string
-    replicas_number    = number
-  })
+variable "labels" {
+  description = "Labels for the Kubernetes StatefulSet to be deployed"
+  type        = map(string)
+  default = {
+    "app"  = "storage"
+    "type" = "table"
+  }
 }
 
-variable "security_context" {
-  description = "security context for MongoDB pods"
+variable "helm_release_name" {
+  description = "Name of the helm chart release, must be shorter than 54 characters"
+  type        = string
+  default     = "mongodb-arm"
```

Comment:
Why this default value?

Reply:
No particular answer, Helm default value would be better ?